app/views/policies/steps/_deployment_options_form.html.erb

```diff
@@ -4,7 +4,7 @@
   <div class="alert alert-info" id="scap-deployment-options-info-banner">
     <span class="pficon pficon-info"></span>
     <strong><%= _('There are significant differences in deployment options.') %></strong>
-    <%= _('Please make sure you understand them by reading our') %> <%=scap_doc_link('#2.3Policydeploymentoptions') %>.
+    <%= _('Please make sure you understand them by reading our') %> <%= link_to(_('documentation'), scap_doc_url('deploying-compliance-policies_security-compliance'), :rel => 'external noopener noreferrer', :target => '_blank') %>.
```

Comment:
Translation wise this isn't great. Other languages may put the word `documentation` at another place, but perhaps we should track it elsewhere.

Reply:
https://projects.theforeman.org/issues/36865